pkg/trace/api/api.go

```diff
@@ -265,7 +265,7 @@ func (r *HTTPReceiver) handleTraces(v Version, w http.ResponseWriter, req *http.
 func (r *HTTPReceiver) processTraces(ts *info.TagStats, traces pb.Traces) {
 	now := time.Now()
 	defer func() {
-		metrics.Timing("datadog.trace_agent.receiver.process_traces_ms", time.Since(now), nil, 1)
+		metrics.Timing("datadog.trace_agent.debug.normalize_ms", time.Since(now), nil, 1)
```

Comment:
Nit: the previous name seemed ok? It matches the function name.

Reply:
I feel there is too much confusion between `process_trace` and `process_traces`. I want to be clear here what we are measuring.